boa_engine/src/object/builtins/jsarraybuffer.rs

```diff
@@ -19,6 +19,20 @@ pub struct JsArrayBuffer {
 
 impl JsArrayBuffer {
     /// Create a new array buffer with byte length.
+    ///
+    /// ```
+    /// # use boa_engine::{
+    /// # object::builtins::JsArrayBuffer,
+    /// # Context,
+    /// # };
+    /// # // Initialize context
+    /// # let context = &mut Context::default();
+    /// // Creates a blank array buffer of n bytes
+    /// let array_buffer = JsArrayBuffer::new(4, context).unwrap();
+    ///
+    /// assert_eq!(array_buffer.take().unwrap(), vec![0_u8; 4]);
+    ///
```

Comment:
```suggestion
    /// # use boa_engine::{
    /// # object::builtins::JsArrayBuffer,
    /// # Context, JsResult,
    /// # };
    /// # // Initialize context
    /// # let context = &mut Context::default();
    /// # fn main() -> JsResult<()> {
    /// // Creates a blank array buffer of n bytes
    /// let array_buffer = JsArrayBuffer::new(4, context)?;
    ///
    /// assert_eq!(array_buffer.take()?, vec![0_u8; 4]);
    /// # }
```
I think it's better to propagate the error instead of unwrapping it.

Reply:
Hmm, sounds good to me. To my knowledge, `unwrap` had been used for all the docs in the wrappers, but I wasn't the biggest fan. I have another branch that I was cleaning up documentation on for a draft proposal. I think I'll probably revamp all of the other docs on that 😄